scripts/_vault_commands.py

```
@@ -34,9 +34,7 @@ def _generate_secret_id(payload: str) -> str:
         raise VaultBootstrapError(msg) from exc
 
-    secret_id = data.get("data", {}).get("secret_id")
-    if not secret_id:
-        msg = "Vault response missing secret_id field"
-        raise VaultBootstrapError(msg)
-
-    return secret_id
+    if secret_id := data.get("data", {}).get("secret_id"):
+        return secret_id
+    else:
+        raise VaultBootstrapError("Vault response missing secret_id field")
```

Ruff (0.14.1)

Avoid specifying long messages outside the exception class

(TRY003)